BreachWarning/qSlicerBreachWarningModule.cxx

```diff
@@ -101,15 +101,15 @@ qSlicerBreachWarningModule::~qSlicerBreachWarningModule()
 //-----------------------------------------------------------------------------
 QString qSlicerBreachWarningModule::helpText()const
 {
-  return "This module can alert the user by color change and sound signal if a tool enters a restricted area."
+  return SlicerBreachWarningModule::tr("This module can alert the user by color change and sound signal if a tool enters a restricted area."

```

Comment:
QObject-based classes should have a static `tr()` method that can be used. Why did you need to add the `SlicerBreachWarningModule::` namespace? What is `SlicerBreachWarningModule` namespace anyway (it does not seem to mach any existing class name)?